src/frontend/context/test/BrevModulContext.test.ts

```diff
@@ -27,7 +27,7 @@ describe('BrevmodulContext', () => {
         });
 
         const behandlingsårsaker = [
-            BehandlingÅrsak.DØDSFALL,
+            BehandlingÅrsak.DØDSFALL_BRUKER,
```

Comment:
I ba-sak tror jeg denne heter DØDSFALL, opprettes det en PR for å endre det i backen?